<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgAccounts</class>
 <widget class="QWidget" name="wdgAccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>211</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbl">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Account list</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkInactivas">
           <property name="text">
            <string>Show inactive accounts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="mqtwDataWithObjects" name="mqtwAccounts" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblTotal">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAccountAdd">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New account</string>
   </property>
   <property name="toolTip">
    <string>New account</string>
   </property>
  </action>
  <action name="actionAccountReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/coins.png</normaloff>:/xulpymoney/coins.png</iconset>
   </property>
   <property name="text">
    <string>Account report</string>
   </property>
   <property name="toolTip">
    <string>Account report</string>
   </property>
  </action>
  <action name="actionAccountDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete account</string>
   </property>
   <property name="toolTip">
    <string>Delete account</string>
   </property>
  </action>
  <action name="actionActive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Is it active?</string>
   </property>
   <property name="toolTip">
    <string>Is it active?</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/transfer.png</normaloff>:/xulpymoney/transfer.png</iconset>
   </property>
   <property name="text">
    <string>Transfer between accounts</string>
   </property>
   <property name="toolTip">
    <string>Transfer between accounts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwDataWithObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
